.pre-commit-config.yaml

```diff
@@ -83,4 +83,4 @@ repos:
         args: ["--quiet"]
         exclude: ".cu"
 
-exclude: ".svg|common/autoware_auto_perception_rviz_plugin"
+exclude: ".svg|control/trajectory_follower|control/trajectory_follower_nodes|common/autoware_auto_dependencies|common/autoware_auto_perception_rviz_plugin|common/osqp_interface|simulator/simple_planning_simulator|planning/parking/freespace_planner|planning/astar_search|planning/costmap_generator|common/rviz_plugins/autoware_auto_perception_rviz_plugin"
```

Comment:
you excludes some unexisted packages.
Please add only common/osqp_interface

Reply:
@tkimura4 
I use the same settings as Proposal So without them it will end up failing again
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/use-autoware-auto-msgs/.pre-commit-config.yaml

Reply:
@taikitanaka3 @tkimura4 Could you create an issue for removing this exclude setting after porting is done?